index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome to Microverse</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+   <h1>Hello Microverse!</h1>
```

Comment:
Great job @asharanjith 🤝

- [ ] [Optional] As stated in the project requirements, kindly add a class and apply styles to this class instead of using the DOM element itself.

Reply:
Thank you.